mpp/docs/Stubbed Projects.md

```diff
@@ -0,0 +1,36 @@
+# Stubbed Projects
```

Comment:
Synced with @igordmn again about it.

Doing so forces all our libraries having these stubs in dependency graph forever. AOSP had similar issue and they use `dependencies { constraints {} }` block to enforce updating related library without depending on it.(Like compose-ui has constraints to compose-foundation).

Using this we might enforce to update some modules without direct dependency. It also will help with K2-K1 compatibility issue